spec/linter_spec.rb

```diff
@@ -63,11 +58,7 @@
     expect(linter.check_stream(invalid_stream)).to be(false)
   end
 
-  it 'should be unhappy with an empty YAML file' do
-    expect(linter.check(spec_data('empty.yaml'))).to be(false)
-  end
-
-  it 'should be unhapy with a YAML file full of spaces' do
-    expect(linter.check(spec_data('spaces.yaml'))).to be(false)
+  it 'should be happy with a YAML file full of spaces' do
+    expect(linter.check(spec_data('spaces.yaml'))).to be(true)
```

Comment:
Can you rename the file and add valid_ to the start of the file name?  Then remove the references to spaces.yaml and empty.yaml from the rakefile.